Calendula/AuthService.cs

```diff
@@ -1,34 +1,64 @@
-﻿using System.Text.Json;
+﻿using Microsoft.Identity.Client;
+using Microsoft.Identity.Client.Extensions.Msal;
+using System.Text.Json;
 
 namespace Calendula
 {
     public class AuthService
     {
-        private readonly string ClientId = "";
-        private HttpClient Client { get; set; } = new HttpClient();
-        public AuthService(string clientId)
+		private const string TokenCacheFileName = "CalendulaTokenCache";
```

Comment:
@plohm12  - the formatting looks a bit off on my screen for this file - can you confirm the formatting looks ok on VS ? Or maybe run it through the formatter? Other than that looks sick

Reply:
yea i see what you mean... i tested the MSAL code in linqpad and copied it into this class file, so i'll have to format it.